src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -972,7 +976,7 @@ protected BoundExpression BindInferredVariableInitializer(DiagnosticBag diagnost
 
             ImmutableArray<BoundExpression> arguments = BindDeclaratorArguments(declarator, localDiagnostics);
 
-            if (kind == LocalDeclarationKind.FixedVariable || kind == LocalDeclarationKind.UsingVariable)
```

Comment:
> || kind == LocalDeclarationKind.UsingVariable [](start = 58, length = 46)

This feels like it's going to regress existing scenarios with `using` statement.

Reply:
Also, don't we want this error? `using string x;` should be illegal. Are we prohibiting it in some other place?